internal/config/config.go

```diff
@@ -92,7 +92,7 @@ var config struct {
 	GCP struct {
 		ProjectID   string `env:"PROJECT_ID" env-default:"" env-description:"GCP service account project id"`
 		JSON        string `env:"JSON" env-default:"e30K" env-description:"GCP service account credentials (base64 encoded)"`
-		DefaultZone string `env:"DEFAULT_ZONE" env-default:"us-east1" env-description:"GCP region when not provided"`
+		DefaultZone string `env:"DEFAULT_ZONE" env-default:"us-central1-a" env-description:"GCP region when not provided"`
```

Comment:
Why? HCCP is deployed in us-east1 I would expect the default to align with that (as well as the AWS default).

Reply:
us-east1 is a region and not a zone is GCP: 
https://cloud.google.com/compute/docs/regions-zones
What zone do you want to be the default?

Reply:
I've not found in docs what zone the image gets built into, but IMHO, we should use the same zone.

Reply:
@adiabramovitch it's https://github.com/osbuild/image-builder/blob/c02a1be7dea2252927b684a06e1feebc47de42b4/templates/image-builder.yml#L232 `us-east4` region for image builder (so zone would be us-east4-a)?

IMHO going forward, we should have a region selection and zone would be selected within that region on the background.

Tho for now I'd be ok to have default to same what image builder has and lets merge :)